apps/molgenis-viz/src/views/charts-home.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <Page>
+    <PageSection :verticalPadding="2">
+      {{ row }}
+      <!-- <DataTableEmx
+        tableId="data-table-test"
+        table="ClinicalData_groupBy"
+        columns="
```

Comment:
remove instead of uncomment ?

Reply:
I left it for local dev/testing purposes, but I can move it to a local file that is ignored